### PR TITLE
RFC: Fix some javadoc issues

### PR DESCRIPTION
**Pull request by rPraml**
When running a maven build, I get
```
[WARNING] Some problems were encountered while building the effective model for io.ebean:ebean-core:jar:14.8.0
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-javadoc-plugin is missing. @ line 238, column 15
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```
Then I get
```
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:3.10.1:javadoc' has not been previously called for the module: 'io.ebean:ebean-api:jar:14.8.0'. Trying to invoke it...
[ERROR] MavenInvocationException: Error when invoking Maven, consult the invoker log file: /home/roland/dev/ebean/ebean-core/target/invoker/maven-javadoc-plugin1392421445.txt
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:3.10.1:javadoc' has not been previously called for the module: 'io.ebean:ebean-core-type:jar:14.8.0'. Trying to invoke it...
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:3.10.1:javadoc' has not been previously called for the module: 'io.ebean:ebean-platform-h2:jar:14.8.0'. Trying to invoke it...
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:3.10.1:javadoc' has not been previously called for the module: 'io.ebean:ebean-platform-postgres:jar:14.8.0'. Trying to invoke it...
[INFO] The goal 'org.apache.maven.plugins:maven-javadoc-plugin:3.10.1:javadoc' has not been previously called for the module: 'io.ebean:ebean-platform-sqlserver:jar:14.8.0'. Trying to invoke it...
```

In the invoker log, there I find 
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.10.1:javadoc (default-cli) on project ebean-api: An error has occurred in Javadoc report generation:
[ERROR] Exit code: 1
[ERROR] /home/roland/dev/ebean/ebean-api/src/main/java/io/ebean/ScriptRunner.java:9: error: self-closing element not allowed
[ERROR]  * <p/>
```

### CODE REVIEW

File: ebean-api/src/main/java/io/ebean/DatabaseBuilder.java [hunk 1]
```diff
@@ -1968,8 +1968,8 @@ default DatabaseBuilder localOnlyL2Cache(boolean localOnlyL2Cache) {
   DatabaseBuilder setLocalOnlyL2Cache(boolean localOnlyL2Cache);
 
   /**
-   * Controls if Ebean should ignore <code>&x64;javax.validation.contstraints.NotNull</code> or
-   * <code>&x64;jakarta.validation.contstraints.NotNull</code>
+   * Controls if Ebean should ignore <code>&#64;javax.validation.contstraints.NotNull</code> or
+   * <code>&#64;jakarta.validation.contstraints.NotNull</code>
    * with respect to generating a <code>NOT NULL</code> column.
```
Comment (rPraml) [inline ebean-api/src/main/java/io/ebean/DatabaseBuilder.java:1973]: this is obviously wrong

File: ebean-core/pom.xml [hunk 2]
```diff
@@ -239,7 +239,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
         <configuration>
-          <doctitle>Ebean 12</doctitle>
+          <doctitle>Ebean 14</doctitle>
           <overview>src/main/java/io/ebean/overview.html</overview>
```
Comment (rPraml) [inline ebean-core/pom.xml:243]: This configuration is not used, when javadoc is called for other modules (`javadoc' has not been previously called for the module: 'io.ebean:ebean-api:jar`)

File: pom.xml [hunk 3]
```diff
@@ -56,6 +56,7 @@
     <ebean-agent.version>14.8.0</ebean-agent.version>
     <ebean-maven-plugin.version>14.8.0</ebean-maven-plugin.version>
     <surefire.useModulePath>false</surefire.useModulePath>
+    <maven-javadoc-plugin.version>3.10.1</maven-javadoc-plugin.version>
```
Comment (rPraml) [inline pom.xml:59]: java11-oss defines 3.7.0 (but both versions have the same problem)

File: pom.xml [hunk 4]
```diff
@@ -102,6 +103,20 @@
...
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <version>${maven-javadoc-plugin.version}</version>
+          <configuration>
+            <additionalOptions>-Xdoclint:none</additionalOptions>
```
Comment (rPraml) [inline pom.xml:114]: The problem is, that there are many javadocs, that uses non-HTML4-compliant tags.
I don't know, what's the best here. Change the HTML5 tags (that obviously work in the browser) to HTML4 tags or disable linting in ebean-paren (or only in specific module)

File: ebean-core/pom.xml (hunk 2 above)
Comment (rbygrave) [reply ebean-core/pom.xml:243]: Yeah, this can be removed really. Good catch.

File: pom.xml (hunk 4 above)
Comment (rbygrave) [reply pom.xml:114]: Yes. disable linting seems reasonable for now.

**Review by rbygrave — APPROVED** [review]

**Pull request merged**